src/main/java/core/basesyntax/Truck.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+public class Truck extends Machine {
+
+    @Override
+    public void doWork() {
+        System.out.println("The truck started working.");
+    }
+
+    @Override
+    public void stopWork() {
+        System.out.println("The truck stopped working.");

```

Comment:
Similarly, the message in the 'stopWork' method should be more informative. Indicate that it is a 'Truck' that has stopped working to provide more context.